attributes/default.rb

```diff
@@ -143,6 +143,9 @@
 
 # Configure Abiquo websockify
 default['abiquo']['websockify']['port'] = 41337
+default['abiquo']['websockify']['api_url'] = 'https://localhost/api'
```

Comment:
Well, we can leave it as-is, as the resolving in the UI makes sense. We allow there to specify an attribute in the node (fqdn, set_fqdn, etc) to be used for the domain part, and I think it is good to keep that.
It does not make sense here, though, so let's keep this as-is.

Future improvements would be for the remote services, ui, websockify, etc, to search for the server node in the Chef Server :)